.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,32 @@
+name: Benchmarks
+
+on: push
+
+jobs:
+  test:
+    name: Send metric over UDP
+    runs-on: ubuntu-18.04
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Setup Ruby
+      uses: actions/setup-ruby@v1
+      with:
+        ruby-version: 2.6
+
+    - name: Install dependencies
+      run: gem install bundler && bundle install --jobs 4 --retry 3
+
+    - name: Run benchmark on branch
+      run: benchmark/send-metrics-to-local-udp-receiver
```

Comment:
Shouldn't we get rid of the test/benchmark/ folder?  It is kind of confusing to have benchmarks in the test folder, but then not being the ones run on CI

Reply:
Yeah, I was going to clean that up in a followup.